pkg/scheduler/framework/types.go

```diff
@@ -227,6 +229,8 @@ type FitError struct {
 const (
 	// NoNodeAvailableMsg is used to format message when no nodes available.
 	NoNodeAvailableMsg = "0/%v nodes are available"
+	// PostFilterMsg is used to format message returned from PostFilterPlugins.
+	PostFilterMsg = "preemption failure: %v"
```

Comment:
it's not a failure... more like it didn't help.

maybe we can just say `preemption:`? Or we could include the word `status`, `result`...?

Reply:
done